week6/vite-project/src/components/CardList.jsx

```diff
@@ -0,0 +1,73 @@
+import { useState, useEffect } from "react";
+import { API_URL } from "./Const";
+import { getTimeAgo } from "./TimeAgo";
+import "../style/CardList.css";
+
+function CardList({ onMouseOver, onMouseOut }) {
+  const [cardLinkData, setCardLinkData] = useState({});
+
+  async function getCardLinkData() {
+    try {
+      const response = await fetch(`${API_URL}/folder`);
+      const linkData = await response.json();
+      setCardLinkData(linkData);
+    } catch (error) {
+      throw new Error("폴더 정보를 가져오는데 실패했습니다.");
+    }
+  }
+
+  function formatDate(value) {
+    const date = new Date(value);
+    return `${date.getFullYear()}. ${date.getMonth() + 1}. ${date.getDate()}`;
+  }
+
+  // function HoverCard() {
+  //   const [isHovered, setIsHovered] = useState(false);
+  //   const handleMouseOver = () => setIsHovered(true);
+  //   const handleMouseLeave = () => setIsHovered(false);
+
+  //   const className = isHovered ? "card card-hovered" : "card";
+
+  //   return (
+  //     <div
+  //       className={className}
+  //       onMouseOver={handleMouseOver}
+  //       onMouseLeave={handleMouseLeave}
+  //     ></div>
+  //   );
+  // }
+
+  useEffect(() => {
+    getCardLinkData();
+  }, []);
+
+  return (
+    <div className="cardlist-section">
+      <div className="cardlist">
+        {cardLinkData?.folder?.links.map(card => (
```

Comment:
links의 요소는 link 인게 자연스럽고, (동료 또는 미래의 내가) 읽고 개선하기 편할거에요
```suggestion
        {cardLinkData?.folder?.links.map(link => (
```